pkg/projects/Microsoft.NETCore.DotNetHost/Microsoft.NETCore.DotNetHostResolver.pkgproj

```diff
@@ -16,6 +16,9 @@
     <Dependency Include="Microsoft.NETCore.DotNetHost">
       <Version>$(HostFullVersion)</Version>
     </Dependency>
+    <Dependency Include="Microsoft.NETCore.DotNetAppHost">
```

Comment:
@gkhanna79, there is a chicken and egg problem of running tests with an existing CLI vs. removing the `dotnet.exe`. Once CLI consumes this change, it would be easier to ingest a new CLI and then make the necessary packaging changes.